index.js

```diff
@@ -46,7 +84,8 @@ client.on('interactionCreate', async interaction => {
         const message = interaction.message;
         if (interaction.customId === 'selectMenu') {
             const cardIndex = Number(interaction.values[0]);
-            await interaction.update({content: 'Card selected', embeds: await generateCardEmbeds(this.gallery, cardIndex), components: []});
+            //console.log(`Card Index Selected: ${cardIndex}`)
+            await interaction.update({content: 'Card selected', embeds: await generateCardEmbeds(this.gallery, cardIndex, true, false, true, true, true, true), components: []});
```

Comment:
Same suggestion here, although I'm not a fan of passing these as hard-coded values.  Then again, there doesn't seem to be a way to pass additional options in `MessageSelectOptionData`, so we'd probably have to pass a different `customId` depending on the type of card search someone did if we want to support the different types.  But anyway, that's not relevant for this PR.

```suggestion
            await interaction.update({content: 'Card selected', embeds: await generateCardEmbeds(this.gallery, cardIndex, true, false, true, false, false, false), components: []});
```

Reply:
I was also thinking on how to pass these in based on the original search type provided by the user.  Left hardcoded for now as we should be able to address this in a future PR.  Changes to the defaults are also good with me here.